cli/demo_direct_call.sh

```diff
@@ -73,26 +72,26 @@ echo "  Bob's incognito account = ${ICGACCOUNTBOB}"
 echo ""
 
 echo "* Issue ${AMOUNTSHIELD} tokens to Alice's incognito account"
-$TIMEOUT ${CLIENT} trusted set-balance ${ICGACCOUNTALICE} ${AMOUNTSHIELD} --mrenclave ${MRENCLAVE} --direct
+${CLIENT} trusted set-balance ${ICGACCOUNTALICE} ${AMOUNTSHIELD} --mrenclave ${MRENCLAVE} --direct
```

Comment:
Removing the timeout means that we potentially wait forever? Or what was the motivation?

Reply:
Yes, that is true. But since this is used as an CI test, I think we should know if no fitting confirmation is issued by the parentchain.

Reply:
So my motiviation: Wait forever so we know something is wrong.